### PR TITLE
Lazy 86 Variant API VS

### PR DESCRIPTION
**Pull request by salyateem13**
Here we add a new chart for API access to apps. We follow a new url convention of api.internal.apps.{{ .Values.istio.ingress.host}} and api.internal.{{ .Values.istio.ingress.host}} for private access via VPN.

If {{ .Values.istio.ingress.public}} is true, then public endpoints are exposed to api.{{ .Values.istio.ingress.host}} and api.apps.{{ .Values.istio.ingress.host}} 

the api.*.apps.{{ .Values.istio.ingress.host}} will be deprecated once a Hosted Zone is setup for api.*.{{ .Values.istio.ingress.host}}. Therefore, api.*.{{ .Values.istio.ingress.host}} will not work when testing this change, only api.*.apps.{{ .Values.istio.ingress.host}} will

### CODE REVIEW

File: charts/variant-api/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+You can reach the application at through vpn-<env>-drivevariant.com
+
+https://api.internal.apps.{{ .Values.istio.ingress.host }}/{{ .Release.Namespace }}/{{ $fullName }}
+
+{{- if .Values.istio.ingress.public }}
+....or 
+https://api.apps.{{ .Values.istio.ingress.host }}/{{ .Release.Namespace }}/{{ $fullName }}
+for public access
+{{- end}}
```
Comment (vibindaniel) [inline charts/variant-api/templates/NOTES.txt:12]: Can you also list the APIs that will not be available for public access?

**Review by vibindaniel — CHANGES_REQUESTED** [review]

**Comment by vibindaniel** [comment]
Let us update the iaac repository to allow api.<domain> to api.apps.<domain>. I'll create a ticket for this.

File: charts/variant-api/templates/NOTES.txt (hunk 1 above)
Comment (salyateem13) [reply charts/variant-api/templates/NOTES.txt:12]: Yep, will get that included

**Comment by salyateem13** [comment]
> Let us update the iaac repository to allow api. to api.apps.. I'll create a ticket for this.

I see you merged that PR. We still need a hosted zone for the cluster before we can remove api.apps.* from the chart, correct?

**Review by vibindaniel — APPROVED** [review]

**Pull request merged**